NekesaH/AssemblyLab1
# Initiliazed variables

### What does this pull request (PR) do?

This is the first part of that question am not sure if its one

### How can this manually be tested
1. Checkout this branch `git checkout q-one-part-one
2. Open the file `lab1.asm` on your `emu8086` program
3. Hit the _emulate_ button. You should see the world around you spinning

### Screenshots (Where necessary)
> If your feature is printing something to the screen or getting input, a screenshot of it **running** would be pretty nice
